Assets/Scripts/GameManager.cs

```diff
@@ -37,13 +54,38 @@ void Update()
                 }
             }
             */
+
+            Debug.Log("click");
+
             Ray ray = Camera.main.ScreenPointToRay(Input.mousePosition);
             RaycastHit hit;
             if (Physics.Raycast(ray, out hit, 100, LayerMask.NameToLayer("Terrain")))
             {
                 Debug.Log(hit.point);
+
+                float x = hit.point.x;
+                float z = hit.point.z;
+
+                int row = (int)(x + width / 2 - .5);
+                int col = (int)(z + height / 2 - .5);
+                Debug.Log("row: " + row + " col: " + col);
+
+                if(makeWetDry == WetDry.wet) {
+                    int old_wet =  board[row, col].moisture;
+                    board[row, col].moisture += 10;
+                    Debug.Log("wetting " + row + ", " + col + " was: " + old_wet + " now: " + board[row, col].moisture);
+
+                }
+                if (makeWetDry == WetDry.dry) {
+                    Debug.Log("drying " + row + ", " + col);
+
+                    board[row, col].moisture -= 10;
+                }
+
+                BoardDisplay display = FindObjectOfType<BoardDisplay>();
```

Comment:
Since the `BoardDisplay` is a component on the same object, you can also do `GetComponent<BoardDisplay> ();`.  I think `FindObjectOfType` searches the entire scene.